eth/v1alpha1/beacon_chain.proto

```diff
@@ -611,12 +611,6 @@ message ActiveSetChanges {
 
     // Indices of validators slashed in the given epoch.
     repeated uint64 slashed_indices = 7;
-
-    // 48 byte validator public keys that have been involuntarily ejected in this epoch.
```

Comment:
They are unused now, but should we use them? (ie. they are just not yet implemented). It seems like nice feature to have from API's stand point

Reply:
What is unused about this? It's not implemented or it's not a relevant feature?